bip-musig2.mediawiki

```diff
@@ -268,16 +285,24 @@ Input:
 ** Return 1
 * Return ''int(hash<sub>KeyAgg coefficient</sub>(L || pk')) mod n''<ref>The key aggregation coefficient is computed by hashing the public key instead of its index, which requires one more invocation of the SHA-256 compression function. However, it results in significantly simpler implementations because signers do not need to translate between public key indices before and after sorting.</ref>
 
-'''''ApplyTweak(Q<sub>i-1</sub>, gacc<sub>i-1</sub>, tacc<sub>i-1</sub>, tweak<sub>i</sub>, is_xonly_t<sub>i</sub>)''''':
-* If ''is_xonly_t<sub>i</sub>'' and ''not has_even_y(Q<sub>i-1</sub>)'':
-** Let ''g<sub>i-1</sub> = -1 mod n''
-* Else: let ''g<sub>i-1</sub> = 1''
-* Let ''t<sub>i</sub> = int(tweak<sub>i</sub>)''; fail if ''t &ge; n''
-* Let ''Q<sub>i</sub> = g<sub>i-1</sub>⋅Q<sub>i-1</sub> + t<sub>i</sub>⋅G''
-** Fail if ''is_infinite(Q<sub>i</sub>)''
-* Let ''gacc<sub>i</sub> = g<sub>i-1</sub>⋅gacc<sub>i-1</sub> mod n''
-* Let ''tacc<sub>i</sub> = t<sub>i</sub> + g<sub>i-1</sub>⋅tacc<sub>i-1</sub> mod n''
-* Return ''(Q<sub>i</sub>, gacc<sub>i</sub>, tacc<sub>i</sub>)''
+===== Applying Tweaks =====
+
+Input:
+* The ''keygen_ctx'': a [[#keygen-context|KeyGen Context]] data structure
+* The ''tweak'': a 32-byte array
+* The tweak mode ''is_xonly_t'': a boolean
+
+'''''ApplyTweak(keygen_ctx, tweak, is_xonly_t)''''':
+* Let ''(Q, gacc, tacc) = keygen_ctx''
+* If ''is_xonly_t'' and ''not has_even_y(Q)'':
+** Let ''g = -1 mod n''
+* Else: let ''g = 1''
```

Comment:
```suggestion
* Else:
** Let ''g = 1''
```

Reply:
fixed

Reply:
You said "fixed" here but you didn't update the branch.

Reply:
Sorry, should be done now.